packages/engine/src/schemas/setting/client-setting.schema.ts

```diff
@@ -155,6 +156,7 @@ export const clientSettingDataSchema = Type.Pick(
     'themeSettings',
     'themeModes',
     'key8thWall',
+    'privacyPolicy',
```

Comment:
Add `privacyPolicy` in `clientSettingQueryProperties` section as well.

Reply:
added!